gst/nnstreamer/tensor_filter/tensor_filter.c

```diff
@@ -512,22 +603,38 @@ gst_tensor_filter_set_property (GObject * object, guint prop_id,
     }
     case PROP_MODEL:
     {
-      const gchar *model_file = g_value_get_string (value);
+      const gchar *model_files = g_value_get_string (value);
+      guint model_num;
 
       if (prop->model_file) {
         gst_tensor_filter_close_fw (self);
         g_free_const (prop->model_file);
         prop->model_file = NULL;
       }
 
-      /* Once configures, it cannot be changed in runtime */
-      g_assert (model_file);
+      if (prop->model_file_sub) {
+        gst_tensor_filter_close_fw (self);
+        g_free_const (prop->model_file_sub);
+        prop->model_file_sub = NULL;
+      }
 
-      silent_debug ("Model = %s\n", model_file);
-      if (!g_file_test (model_file, G_FILE_TEST_IS_REGULAR)) {
-        GST_ERROR_OBJECT (self, "Cannot find the model file: %s\n", model_file);
-      } else {
-        prop->model_file = g_strdup (model_file);
+      /* Once configures, it cannot be changed in runtime */
```

Comment:
If I understand correctly, I think you can use `gst_element_get_state()` and reject configurations in RUNNING or other states

Reply:
I agree with your opinion. However, it's not the purpose of this pr. I will make it better at the next pr. to remember, I added `todo` for it.